docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc

```diff
@@ -78,6 +78,8 @@ When SCRAM is enabled, {agent} uses the link:https://en.wikipedia.org/wiki/Salte
 * SCRAM-SHA-256 - uses the SHA-256 hashing function
 * SCRAM-SHA-512 - uses the SHA-512 hashing function
 
+To prevent unauthorized access your Kafka password is stored as a <<agent-policy-secret-values,secret value>>, separate from other policy details. While secret storage is recommended, you can choose to override this setting and store the password as plain text in the agent policy definition. Secret storage requires {fleet-server} version 8.12 or higher.
```

Comment:
Could the link to "Policy secret value" be confusing? While the latter also store values as secrets, it's not strictly speaking related information. For the same reason, I would suggest removing ", separate from other policy details". (Same comment for Kafka and Logstash SSL keys 🙂).

Reply:
Definitely. I've removed the link and the "separate from other policy details" text. The change is in both the Kafka and the Logstash sections.